power_grid_model_c/power_grid_model/include/power_grid_model/component/source.hpp

```diff
@@ -42,7 +42,9 @@ class Source : public Appliance {
         y0_ref_ = y1_ref_ / z01_ratio;
     }
 
-    // getter for calculation param, y_ref
+    // getter for y0 y1 ref
+    std::pair<DoubleComplex, DoubleComplex> get_y0_y1() const { return {y0_ref_, y1_ref_}; }
```

Comment:
As I said before, put this logic directly into `template <symmetry_tag sym> ComplexTensor<sym> math_param()`